tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to the Advanced Research Computing (ARC)\n",
+    "\n",
+    "Open a terminal and remotely connect to ARCUS using ssh (secure shell), using your username:\n",
+    "\n",
+    "`% ssh -X username@oscgate.arc.ox.ac.uk`\n",
+    "\n",
+    "When prompted, enter your password and hit Enter.\n",
+    "Now, you need to connect to the arcus-htc cluster, so in your command line type:\n",
+    "\n",
+    "`% ssh -X arcus-htc`\n",
+    "\n",
+    "If you get a message like this one:\n",
+    "\n",
+    "The authenticity of host 'arcus-htc (10.137.128.21)' can't be established.\n",
+    "RSA key fingerprint is d1:83:21:b1:f2:bd:8f:e7:5d:cd:74:d1:73:b9:70:7a.\n",
+    "Are you sure you want to continue connecting (yes/no)? \n",
+    "\n",
+    "Type \n",
+    "\n",
+    "`% yes`\n",
+    "\n",
+    "You can now check the path to the directory you are at, by typing:\n",
+    "\n",
+    "`% pwd`\n",
+    "\n",
+    "It should be your home directory (/home/username). This is where you will work from, you will need to create some subdirectories to organise your files. First creatge a directory for your setup and move to it, as it will be your first working directory:\n",
+    "\n",
+    "`% mkdir setup`\\\n",
+    "`% cd setup`\n",
+    "\n",
+    "Now, you need to load GROMACS, which is the simulation software package that you will utilise:\n",
```

Comment:
Can we move this under the next section, renaming the next section as "The GROMACS Molecular Dynamics Engine"?

That way you have this nice introduction that says "this is what you'll be using", and then you can go "in ARCUS to use gromacs you should do the following".

Reply:
Done